engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -511,6 +532,16 @@ public JSONObject getLogicalOperators() throws Exception
     nodeList.operators = dagManager.getLogicalOperatorInfoList();
     return new JSONObject(objectMapper.writeValueAsString(nodeList));
   }
+  
+  @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES)
+  @Produces(MediaType.APPLICATION_JSON)
+  public JSONObject getLogicalModules(@QueryParam("flatten") String flatten) throws Exception
```

Comment:
tried it, this does not works too. 

```java
  public JSONObject getLogicalModules(@QueryParam("flatten") Boolean flatten) throws Exception
  {
    LogicalModulesInfo nodeList = new LogicalModulesInfo();
    nodeList.modules = dagManager.getLogicalModuleInfoList(BooleanUtils.isTrue(flatten));
    return new JSONObject(objectMapper.writeValueAsString(nodeList));
  }
```

Tried issuing following requests
```
/modules?flatten=true
/modules?flatten=True
/module?flatten=false
/module?flatten
```
In all the cases value of flatten was false.